### ISSUE TITLE
Implement exists function on prisma client

### ISSUE THREAD

**Issue by greguintow** [prisma/prisma-client-js#474]
This function have on prisma-binding, it's very helpful, so I thought would be a good idea to insert on the new prisma client.
This functions returns a boolean, you are able to set as many filters (like where from findMany) as you want, if all pass is going to return true, if one fail it's going to return false. There are some images to help understand on this issue: #469.

**Comment by maticzav** [comment]
I agree! `exists` function is very beneficial for implementing permissions where you want to know whether something exists in the backend with specific criteria.

**Comment by greguintow** [comment]
Hello?
